dotCMS/src/main/java/com/dotcms/publisher/pusher/PushUtils.java

```diff
@@ -76,8 +76,10 @@ private static void addFilesToCompression(TarArchiveOutputStream taos, File file
 				if (file.isFile()) {
 			        // Add the file to the archive
 					BufferedInputStream bis = new BufferedInputStream(new FileInputStream(file));
-					IOUtils.copy(new FileInputStream(file), taos);
+					FileInputStream in = new FileInputStream(file);
```

Comment:
Only "in" is used, and by using "in" and not "bis" we are losing the buffer that BufferedInputStream does, however, IOUtils.copy does buffering internally (4096 vs 8192 of BufferedInputStream)